package.json

```diff
@@ -0,0 +1,37 @@
+{
+  "name": "kuzzle-vault",
+  "version": "1.0.0",
+  "description": "Encrypt and decrypt secrets with Kuzzle Vault",
+  "main": "./index.js",
+  "scripts": {
+    "test": "npm run --silent lint && npm run unit-testing",
+    "unit-testing": "nyc --reporter=text-summary --reporter=lcov mocha",
+    "codecov": "codecov",
+    "lint": "eslint --max-warnings=0 ./ ./lib ./test"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/kuzzleio/kuzzle-vault.git"
+  },
+  "author": "The Kuzzle Team <support@kuzzle.io>",
+  "license": "Apache-2.0",
+  "bugs": {
+    "url": "https://github.com/kuzzleio/kuzzle-vault/issues"
+  },
+  "homepage": "https://github.com/kuzzleio/kuzzle-vault#readme",
+  "dependencies": {
+    "assert": "^2.0.0",
```

Comment:
I don't see that module used. Furthermore, this module aims to provide [Node.js assert](https://nodejs.org/dist/latest-v10.x/docs/api/assert.html) to browsers, so I don't think we need it.